### ISSUE TITLE
About Syncnet loss

### ISSUE THREAD

**Issue by MontaEllis** [Rudrabha/Wav2Lip#57]
Hi, I'm glad to see your new project called wav2lip!
Here I have some questions about sync loss.
You can see https://github.com/Rudrabha/Wav2Lip/blob/d7259e531bf984c6af1eef6ea4511923694c1ac6/hq_wav2lip_train.py#L196, the inputs of sync loss are mel and images. The mel is (bs * 80 * 16), well the images should be (bs * 3 * H * W). The sync loss should take 5 pieces of images and the corresponding mel to calculate. Your code **torch.cat([g[:, :, i] for i in range(syncnet_T)], dim=1)** does not do the what sync loss want!
I would be very sorry if I have any wrong understanding!

**Comment by prajwalkr** [comment]
the shape of g before that line is: `(B, 3, T, H/2, W)`. The shape of g after that line is `(B , 3 * T, H/2, W)`, `T=5`.
